src/Crypto/WebAuthn/Metadata/FidoRegistry.hs

```diff
@@ -36,7 +35,12 @@ data UserVerificationMethod
   | USER_VERIFY_NONE
   | USER_VERIFY_ALL
   deriving (Show, Eq, Generic)
-  deriving (Aeson.FromJSON, Aeson.ToJSON) via EnumJSONEncoding "USER_VERIFY_" UserVerificationMethod
+
+instance Aeson.FromJSON UserVerificationMethod where
+  parseJSON = Aeson.genericParseJSON $ enumJSONEncodingOptions "USER_VERIFY_"
+
+instance Aeson.ToJSON UserVerificationMethod where
+  toJSON = Aeson.genericToJSON $ enumJSONEncodingOptions "USER_VERIFY_"
```

Comment:
I think we should also implement the `toEncoding` method for all `ToJSON` instances, see https://hackage.haskell.org/package/aeson-2.0.3.0/docs/Data-Aeson.html#v:toEncoding

Reply:
After discussion, many duplication would be needed. We can't use `Data.Aeson.TH` because it doesn't work with GADTs, and there is no use in using `Data.Aeson.GADT.TH` because it doesn't use `toEncoding`.